### PR TITLE
Set shared-volume and meta-volume options

### PR DESCRIPTION
**Pull request by sac**

### CODE REVIEW

File: playbooks/georep-init-slave-pem-setup.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@
   gather_facts: no
 
   tasks:
+  - name: Enable shared storage for geo-rep slave volume
```
Comment (aravindavk) [inline playbooks/georep-init-slave-pem-setup.yml]: Shared volume not required in Slave

Comment (sac) [reply playbooks/georep-init-slave-pem-setup.yml]: @aravindavk I followed the document available at https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.4/html-single/administration_guide/index#idm139715244228960

10.3.4 point number 3:
Enable shared storage for master and slave volumes

Should I raise a documentation bug to change that?

Comment (aravindavk) [reply playbooks/georep-init-slave-pem-setup.yml]: @kotreshhr do you know any reason to enable in Slave side?

Comment (sunnyku) [reply playbooks/georep-init-slave-pem-setup.yml]: I guess its due to snap scheduler.

**Comment by sac** [comment]
@aravindavk I've updated the patch.

**Review by aravindavk — APPROVED** [review]
LGTM

**Comment by sac** [comment]
@aravindavk thanks for the review.

**Pull request merged**